lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -78,11 +106,16 @@ namespace Intl
     void SetNumberFormatGroupingUsed(_In_ IPlatformAgnosticResource *resource, _In_ const bool isGroupingUsed);
 
     template <typename T>
-    const char16 *FormatNumber(IPlatformAgnosticResource *formatter, const T val, const NumberFormatStyle formatterToUse, const NumberFormatCurrencyDisplay currencyDisplay, const char16 *currencyCode);
+    const char16 *FormatNumber(IPlatformAgnosticResource *formatter, const T val, const NumberFormatStyle formatterToUse,
+        const NumberFormatCurrencyDisplay currencyDisplay, const char16 *currencyCode);
 
-    bool ResolveLocaleLookup(_In_z_ const char16 *locale, _Out_ char16 *resolved);
+    bool IsLocaleAvailable(_In_z_ const char *locale);
     bool ResolveLocaleBestFit(_In_z_ const char16 *locale, _Out_ char16 *resolved);
-    size_t GetUserDefaultLanguageTag(_Out_ char16* langtag, _In_ size_t cchLangtag);
+    size_t GetUserDefaultLocaleName(_Out_ char16* langtag, _In_ size_t cchLangtag);
```

Comment:
>LocaleName [](start = 25, length = 10)

Is there a comment somewhere documenting the convention of using LocaleName versus LanguageTag?

Reply:
So that I wouldnt need #if windows localeName #else languageTag. Thats all.